frontend/projects/upgrade/src/app/core/feature-flags/store/feature-flags.effects.ts

```diff
@@ -311,7 +311,7 @@ export class FeatureFlagsEffects {
       filter(({ featureFlagId }) => !!featureFlagId),
       switchMap(({ featureFlagId }) =>
         this.featureFlagsDataService.exportFeatureFlagsDesign(featureFlagId).pipe(
-          map((data) => {
+          map((data: FeatureFlag) => {
```

Comment:
Instead of typing `data` can you type `exportFeatureFlagsDesign`

Reply:
It was an old code of Khanjan, I just added the type `FeatureFlag`. Will update the variable name `data`. 👍